getting_started.md

```diff
@@ -0,0 +1,109 @@
+
+## Download the data
+The workloads in this benchmark use 6 different datasets across 8 workloads. You may choose to download some or all of the datasets as you are developing your submission, but your submission will be scored across all 8 workloads. For instructions on obtaining and setting up the datasets see [datasets/README](https://github.com/mlcommons/algorithmic-efficiency/blob/main/datasets/README.md#dataset-setup).
+
+
+## Develop your submission
+To develop a submission you will write a python module containing your optimizer algorithm. Your optimizer must implement a set of predefined API methods for the initialization and update steps.
+
+### Set up your directory structure (Optional)
+
+1. Make a submissions subdirectory to store your submission modules e.g. `algorithmic-effiency/my_submissions`.
+2. Make an experiment directory to store your experiment results e.g. `~/my_experiments`.
+
+### Coding your submission
+#TODO(kasimbeg): If we want use this submission_template script where should we store it in the repo?
+
+You can find examples of sumbission modules under `algorithmic-efficiency/baselines` and `algorithmic-efficiency/reference_algorithms`. \
+A submission for the external ruleset will consist of a submission module and a tuning search space definition.
+1. Clone the submission_template.py script into your submissions directory e.g. in `algorithmic-efficiency/my_submissions`.
+2. Implement at least the methods in the submission_template. Feel free to use helper functions and/or modules as you see fit.
+3. Add a tuning configuration e.g. `tuning_search_space.json` file to your submission directory. For the tuning search space you can either:
+    1. Define the set of feasible points by defining a value for "feasible_points" for the hyperparameters:
+        ```
+        {
+            "learning_rate": {
+                "feasible_points": 0.999
+                },
+        }
+        ```
+        For a complete example see [tuning_search_space.json](https://github.com/mlcommons/algorithmic-efficiency/blob/main/reference_algorithms/target_setting_algorithms/imagenet_resnet/tuning_search_space.json).
+
+    2. Define a range of values for halton sampling by specifing a "min", "max" and "scaling" 
+        keys for the hyperparameter:
+        ```
+        {
+            "weight_decay": {
+                "min": 5e-3, "max": 1.0, "scaling": "log"
+                }
+        }
+        ```
+        For a complete example see[tuning_search_space.json](https://github.com/mlcommons/algorithmic-efficiency/blob/main/baselines/nadamw/tuning_search_space.json). 
+
+
+## Run your submission
+TODO(kasimbeg): Add script to run all workloads?
```

Comment:
I don't think we need this, because presumably submitters should decide how they need to parallelize their compute (across all workloads, across datasets, all workloads on the same machine in a bash loop, etc), and they can wire that up (idk how much value there is in providing a default .sh that loops over all workloads, because idk how often that setup will be used because it will be a very long running job).

Reply:
Ok sg